### PR TITLE
Rover descriptions

### PR DESCRIPTION
**Pull request by tom-bp**
Created descriptions of the 3 rovers on the front page, with photos using flex boxes

### CODE REVIEW

File: src/frontend/pages/home/Home.js [hunk 1]
```diff
@@ -1,6 +1,11 @@
 import React, { useEffect, useState } from "react";
 import * as QuotationsApi from "../../api/QuotationsApi";
 import "./Home.css";
+const curiosity = "/curiosity.jpg";
+const opportunity = "/oppotunity.jpg";
+const spirit = "/spirit.jpg";
+
+// import NavBar from "../../components/NavBar";
```
Comment (Kenneth-Siu) [inline src/frontend/pages/home/Home.js]: The intent behind this is really good - to mark out where the navbar will go in the future - but when it comes to the practical side, it often ends up being that putting in comments has the potential to be more harmful than helpful. For example, if the navbar is delayed, or implemented another way, or something else, then the comment would be at best outdated, and at worst misleading.

So let's not have these comments and let it be determined by whoever's working on the navbar ticket (even if it's you).

File: src/frontend/pages/home/Home.js [hunk 2]
```diff
@@ -1,6 +1,11 @@
 import React, { useEffect, useState } from "react";
 import * as QuotationsApi from "../../api/QuotationsApi";
 import "./Home.css";
+const curiosity = "/curiosity.jpg";
+const opportunity = "/oppotunity.jpg";
```
Comment (Kenneth-Siu) [inline src/frontend/pages/home/Home.js]: Whoops misspelled "/opportunity.jpg". (Also need to update the file name)

File: src/frontend/pages/home/Home.js [hunk 3]
```diff
@@ -10,10 +15,78 @@ export default function Home() {
...
+                            <div className="text">
+                                <h3>Curiosity Rover</h3>
+
+                                <p1>
+                                    Curiosity is the largest and most capable rover ever sent to Mars. Curiosity is a car-sized Mars rover designed to
+                                    explore the Gale crater on Mars as part of NASA's Mars Science Laboratory (MSL) mission. Curiosity was launched
+                                    from Cape Canaveral (CCAFS) on 26 November 2011, at 15:02:00 UTC and landed on Aeolis Palus inside Gale crater on
+                                    Mars on 6 August 2012, 05:17:57 UTC.
+                                    <br />
+                                    <br />
```
Comment (Kenneth-Siu) [inline src/frontend/pages/home/Home.js]: A bit of HTML best practices:

We want our HTML to be _structural_, and to let our _styling_ be done by the CSS.

`<br />` elements are only there to space things, and are therefore an example of trying to add _styling_ to our HTML... which we don't want.

What we'll want to do is to have the two paragraphs in their own `<p>` elements, and then add a margin to the bottom of each paragraph.

File: src/frontend/pages/home/Home.js [hunk 4]
```diff
@@ -10,10 +15,78 @@ export default function Home() {
...
+                    <h2>Quotation of the day</h2>
+                    <p className="quotation">{quotation?.text}</p>
+                    <p className="attribution">—{quotation?.attribution}</p>
+
+                    <div className="flex-container">
+                        <div>
+                            <div className="text">
+                                <h3>Curiosity Rover</h3>
+
+                                <p1>
```
Comment (Kenneth-Siu) [inline src/frontend/pages/home/Home.js]: Did you mean `<p>`?

**Review by Kenneth-Siu — CHANGES_REQUESTED** [review]
Looks good! Almost there, a few thoughts remaining about best practices

File: src/frontend/pages/home/Home.js (hunk 4 above)
Comment (tom-bp) [reply src/frontend/pages/home/Home.js]: I think i deliberately made it p1 as when i started out i was gonna have the different text aligned in different ways so wanted to differentiate the text, but i never changed it after, would you like me to change it?

File: src/frontend/pages/home/Home.js (hunk 3 above)
Comment (tom-bp) [reply src/frontend/pages/home/Home.js]: okay thats fine, i used break points as i wanted a space between them, will add margin instead

File: src/frontend/pages/home/Home.js (hunk 2 above)
Comment (tom-bp) [reply src/frontend/pages/home/Home.js]: will change this

File: src/frontend/pages/home/Home.js (hunk 1 above)
Comment (tom-bp) [reply src/frontend/pages/home/Home.js]: yes i forgot to remove them, will get it changed

**Review by Kenneth-Siu — APPROVED** [review]
Nice, looks much better!

**Comment by Kenneth-Siu** [comment]
Careful with the merge! Take your time and resolve the conflicts :)

**Pull request merged**